Makefile

```diff
@@ -17,6 +17,13 @@ $(THRIFT_BUILDDIR)/baseplate/thrift/baseplate.thrift_buildstamp: baseplate/thrif
 	cp -r $(THRIFT_BUILDDIR)/$</baseplate/thrift/ baseplate/
 	touch $@
 
+$(THRIFT_BUILDDIR)/tests/integration/test.thrift_buildstamp: tests/integration/test.thrift
+	@echo SPECIAL $< $@
```

Comment:
All make-isms. [@ suppresses echoing the "recipe"](https://www.gnu.org/software/make/manual/html_node/Echoing.html#index-_0040-_0028in-recipes_0029). The $< and $@ are [make automatic variables](https://www.gnu.org/software/make/manual/html_node/Automatic-Variables.html).